samples/SocketsSample/wwwroot/hubs.html

```diff
@@ -130,7 +130,7 @@ <h1 id="head1"></h1>
 
 click('groupmsg', event => {
     let groupName = getText('target');
-    let message = getText('messages');
+    let message = getText('message');
```

Comment:
Alright. I'll go through quickly and give more specific ids to the message inputs.